aea/protocols/dialogue/base.py

```diff
@@ -1253,13 +1276,18 @@ def _load_incomplete_dialogues_labels(self, collection: SyncCollection) -> None:
 
     def _load_dialogues(self, collection: SyncCollection) -> Iterable[Dialogue]:
         """Load dialogues from collection."""
+        if not collection:  # pragma: nocover
+            return
```

Comment:
None return type not accounted for in typing

Reply:
in this case it's the end of the generator, so it's ok